letsencrypt/cli.py

```diff
@@ -512,18 +519,19 @@ def install(args, config, plugins):
 
 def revoke(args, config, unused_plugins):  # TODO: coop with renewal config
     """Revoke a previously obtained certificate."""
+    # For user-agent construction
```

Comment:
Tests for this function would be nice since the majority of the lines changed. I had some on my old absolute paths [PR](https://github.com/letsencrypt/letsencrypt/blob/273e08275b910f31a1967d91e77bf80df2513788/letsencrypt/tests/cli_test.py#L235) you should be able to use with little to no modification.